cmd/osv-scanner/main.go

```diff
@@ -3,11 +3,10 @@ package main
 import (
 	"errors"
 	"fmt"
-	"io"
-	"os"
-
 	"github.com/google/osv-scanner/internal/output"
 	"github.com/google/osv-scanner/pkg/osvscanner"
+	"io"
```

Comment:
nit: revert import move

Reply:
according to the linter this change is required - I'll re-check later in case I've missed something

Reply:
ah right ok apparently I missed out the newline when I checked earlier